core/src/main/java/org/wildfly/extras/creaper/core/online/SslOptions.java

```diff
@@ -174,6 +177,14 @@ public Builder trustStoreType(KeyStoreType trustStoreType) {
             return this;
         }
 
+        /**
+         * Errors during hostname verification will be ignored.
+         */
+        public Builder turnOffHostnameVerification() {
```

Comment:
Actually, the entire concept of verifying a hostname seems to be only related to HTTPS. (You know I don't really understand security, do you? :-) )

Which kinda raises a question about HTTPS_REMOTING, where the HTTPS endpoint is used for upgrade... but no, let's not go there.

The only thing I'd ask for is that this is documented. Something like: "This method only makes sense for the {@link ManagementProtocol#HTTPS HTTPS} management protocol. There's no such thing as hostname verification for the native management protocols."